public/app/plugins/datasource/loki/queryHints.test.ts

```diff
@@ -98,17 +119,69 @@ describe('getQueryHints', () => {
     };
 
     it('suggest level renaming when no level label', () => {
-      expect(getQueryHints('{job="grafana"', [createSeriesWithLabel('lvl')])).toMatchObject([
-        { type: 'ADD_JSON_PARSER' },
-        { type: 'ADD_LOGFMT_PARSER' },
-        { type: 'ADD_LEVEL_LABEL_FORMAT' },
-      ]);
+      expect(getQueryHints('{job="grafana"', [createSeriesWithLabel('lvl')])).toEqual(
+        expect.arrayContaining([expect.objectContaining({ type: 'ADD_LEVEL_LABEL_FORMAT' })])
```

Comment:
How can I trigger this suggestion in the builder?

Reply:
`And for that hint, we check if some label has level-like name - e.g. lvl , logLevel - and if so, we suggest user to rename that label to level - Ivana` so i assume the only way trigger this is to have logs that include one of the following labels "lvl" or "logLevel".

Reply:
i wasnt able to get this to trigger because i use grafana/tns for development and the label is already called "level"

Reply:
We can rename in devenv loki `level` here https://github.com/grafana/grafana/blob/main/devenv/docker/blocks/loki/data/data.js#L77 to `lvl`, so will be able to test it more easily.